package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simplenote",
-  "version": "2.0.0-rc3",
+  "version": "2.0.0-rc5",
```

Comment:
The proper conflict resolution would have been `rc3`, but I thought it appropriate to keep the version in sync between `package.json` and `package-lock.json`.